build-parent/pom.xml

```diff
@@ -487,6 +487,8 @@
                                 <operatorAndVersion>~${version.sedona}</operatorAndVersion>
                             </packageDefinition>
                         </managedDependencies>
```

Comment:
I: Turning off failures the new linting in habushu 2.14.0 so it doesn't stop the build.  we have another ongoing ticket(s) to resolve these linting issues and turn these back to `true`